rmgpy/molecule/draw.py

```diff
@@ -199,7 +199,7 @@ def draw(self, molecule, file_format, target=None):
                     self.molecule.remove_atom(atom)
             self.symbols = ['CO']
             self.molecule.atoms[0].charge = 0  # don't label the C as - if you're not drawing the O with a +
-            self.coordinates = np.array([[0, 0]], np.float64)
+            self.coordinates = np.array([[0, 0]], float)
```

Comment:
I was tired of looking at warnings when running unit tests over and over, I got it to a point where there are only a few raised. Some of the remainder reference Cython function that expect numpy float 64 rather than Python's float, which is why those were left. I may have missed a couple elsewhere.

Reply:
I think I understand now.

The warnings were "_DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here._"
and the fix was replace `np.float` with `float`.  I did the last stragglers in bb6788a4f3656d81e3a8cad72f7d96f355d16343

Converting `np.float64` to `float` I think was unnecessary, but as far as I can tell is harmless (we'll still get double precision, and have mostly done it when typing arrays which will end up using numpy's array scalar equivalent anyway). For consistency I did the last few in ed754dff48a34dc977c9ec311e29bef727a9037e.

Cythonized cdefs using np.float64_t I have left alone.